internal/app/cli.go

```
@@ -218,32 +247,51 @@ func (c *cli) readState(s *state) error {
 	os.RemoveAll(tempFilesDir)
 	_ = os.MkdirAll(tempFilesDir, 0o755)
 
+	if len(c.spec) > 0 {
+
+		sp := new(StateFiles)
+		sp.specFromYAML(c.spec)
+
+		for _, val := range sp.StateFiles {
+			fo := fileOption{}
+			fo.name = val.Path
+			fo.priority = val.Priority
+			checkSpecValid(fo.name)
```

The returned error is not being checked.